toolkit/docs/building/building.md

```diff
@@ -487,6 +490,20 @@ sudo make hydrate-rpms PACKAGE_ARCHIVE=./rpms.tar.gz
 
 > Build the go tools from source as needed.
 
+#### `REFRESH_WORKER_CHROOT=...`
+
+##### `REFRESH_WORKER_CHROOT=`**`n`**
+
+> If exists, don't attempt to rebuild the worker chroot, even if the its build script, its manifest, or the packages it consists of have been modified in the local repository.
```

Comment:
```suggestion
> If exists, don't attempt to rebuild the worker chroot, even if it is the build script, its manifest, or the packages it consists of have been modified in the local repository.
```

Reply:
Another typo, thank you! Removed "the".:)